packages/Interface/src/DTK_C_API.h

```diff
@@ -146,6 +146,19 @@ extern void DTK_finalize();
 
 /**@}*/
 
+/** \brief Get DTK error message.
+ *
+ * All DTK functions set \c errno error code upon completion. If DTK function
+ * fails, the code is nonzero. This function provides a way to get the the
+ * error message associated with the error code. If the error code is unknown,
+ * DTK returns a string stating that.
+ *
+ * \param err Error number (typically, errno)
```

Comment:
> Should we assume that everybody will know what errno is?

I hope so. We could point to some documentation somewhere.



> Regardless we should document that we set its value in other functions from the C API.

This is already said in the description:
```All DTK functions set \c errno error code upon completion.```